src/js/components/DataClearFilters/stories/Simple.js

```diff
@@ -0,0 +1,66 @@
+import React, { useContext } from 'react';
+
+import {
+  Box,
+  Data,
+  DataClearFilters,
+  DataContext,
+  DataFilter,
+  DataFilters,
+  DataSearch,
+  DataSummary,
+  DataTable,
+  SelectMultiple,
+  Toolbar,
+} from 'grommet';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Simple = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box pad="medium">
+    <Data
+      data={DATA}
+      total={DATA.length}
+      properties={{
+        location: { label: 'Location' },
+      }}
+    >
+      <DataToolbar />
+      <DataSummary />
+      <DataTable columns={columns} />
+    </Data>
+  </Box>
+  // </Grommet>
+);
+
+const DataToolbar = () => {
+  const { filteredTotal, total } = useContext(DataContext);
+
+  return (
+    <Toolbar align="end">
+      <DataSearch placeholder="Search" />
+      <DataFilters updateOn="change">
+        <DataFilter
+          property="location"
+          // override theme margin to align with search + filter
+          contentProps={{ margin: { bottom: 'none', top: 'xsmall' } }}
```

Comment:
feels like we could improve here with when the label is used then adjust margin so a user does not have to mess with passing something like this for `margin`

Reply:
Yeah I think that could be an enhancement in future. Kind of tricky in terms of how we might handle because we don't always know how someone's theme is set up/what they'd want to override or not.

At least for MVP, someone can do this but agree it's not ideal.